rune/libenclave/internal/runtime/pal/loader.go

```diff
@@ -10,6 +10,7 @@ import "C"
 
 import (
 	"unsafe"
+	"github.com/sirupsen/logrus"
```

Comment:
use `go fmt` before commiting your codes and you can see "github.com/sirupsen/logrus" is above "unsafe"

Reply:
Formatted already.